{"kind":"symbol","sections":[],"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"invalidCallback","kind":"identifier"}],"symbolKind":"case","title":"SignInWithAppleError.invalidCallback","roleHeading":"Case","role":"symbol","modules":[{"name":"RxFireAuth"}],"externalID":"s:10RxFireAuth20SignInWithAppleErrorO15invalidCallbackyA2CmF"},"schemaVersion":{"patch":0,"major":0,"minor":3},"abstract":[{"type":"text","text":"A callback has been invoked without a nonce."}],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"invalidCallback"}]}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError\/invalidCallback"},"hierarchy":{"paths":[["doc:\/\/RxFireAuth\/documentation\/RxFireAuth","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/rxfireauth\/signinwithappleerror\/invalidcallback"]}],"references":{"doc://RxFireAuth/documentation/RxFireAuth/SignInWithAppleError/invalidCallback":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"invalidCallback","kind":"identifier"}],"title":"SignInWithAppleError.invalidCallback","type":"topic","role":"symbol","kind":"symbol","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError\/invalidCallback","url":"\/documentation\/rxfireauth\/signinwithappleerror\/invalidcallback","abstract":[{"type":"text","text":"A callback has been invoked without a nonce."}]},"doc://RxFireAuth/documentation/RxFireAuth":{"title":"RxFireAuth","type":"topic","role":"collection","kind":"symbol","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth","url":"\/documentation\/rxfireauth","abstract":[]},"doc://RxFireAuth/documentation/RxFireAuth/SignInWithAppleError":{"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"SignInWithAppleError"}],"title":"SignInWithAppleError","navigatorTitle":[{"kind":"identifier","text":"SignInWithAppleError"}],"type":"topic","role":"symbol","kind":"symbol","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError","url":"\/documentation\/rxfireauth\/signinwithappleerror","abstract":[{"type":"text","text":"Errors thrown by "},{"code":"SignInWithAppleHandler","type":"codeVoice"},{"type":"text","text":" instances."}]}}}